testing/tests/loops.rs

```diff
@@ -67,3 +67,39 @@ fn test_for_range() {
         "foo (first)\nfoo (last)\nbar\nbar\nfoo\nbar\nbar\n"
     );
 }
+
+#[derive(Template)]
+#[template(source = "{% for i in [1, 2, 3] %}{{ i }}{% endfor %}", ext = "txt")]
+struct ForArrayTemplate {}
```

Comment:
Note this one is similar to [`ArrayTemplate` in `simple.rs`][arr]. Though I feel like `simple.rs` is more of a "showcasing" of features and semantics, so should be keep both or remove `ForArrayTemplate`?

[arr]: https://github.com/djc/askama/blob/810d5ad5067085cc5e4be19903b5804b1ff13430/testing/tests/simple.rs#L252-L263

Reply:
Let's keep both.